README.Rmd

```diff
@@ -9,30 +9,91 @@ output:
 
 ```{r, echo = FALSE}
 knitr::opts_chunk$set(
-  collapse = TRUE,
-  comment = "#>",
-  fig.path = "figs/README-"
+    collapse = TRUE,
+    comment = "#>",
+    fig.path = "figs/"
 )
 ```
 
 # MoMA: Modern Multivariate Analysis
 
-[![Travis-CI Build Status](https://api.travis-ci.com/michaelweylandt/MoMA.svg)](https://travis-ci.com/michaelweylandt/MoMA) [![License: GPL v2](https://img.shields.io/badge/License-GPL%20v2-brightgreen.svg)](https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html) [![CRAN_Status_Badge](http://www.r-pkg.org/badges/version/MoMA)](https://cran.r-project.org/package=MoMA) [![Project Status: WIP – Initial development is in progress, but there has not yet been a stable, usable release suitable for the public.](http://www.repostatus.org/badges/latest/wip.svg)](http://www.repostatus.org/#wip)
+<!-- badges: start -->
+[![Travis-CI Build Status](https://api.travis-ci.com/michaelweylandt/MoMA.svg)](https://travis-ci.com/michaelweylandt/MoMA) 
+[![License: GPL v2](https://img.shields.io/badge/License-GPL%20v2-brightgreen.svg)](https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html) [![CRAN_Status_Badge](http://www.r-pkg.org/badges/version/MoMA)](https://cran.r-project.org/package=MoMA) 
+[![Project Status: WIP – Initial development is in progress, but there has not yet been a stable, usable release suitable for the public.](http://www.repostatus.org/badges/latest/wip.svg)](http://www.repostatus.org/#wip)
+<!-- badges: end -->
 
-**TODO -- PACKAGE TEXT GOES HERE**
+MoMA is a penalized SVD framework that supports a wide range of sparsity-inducing penalties. For a matrix _X_, MoMA gives the solution to the following optimization problem:
+
+<center><img src="man/figures/moma-formula.svg" style="width: 500; max-width: 100%; height: auto" title="MoMA formula" /><br></center>
+
+The penalties (the _P_ functions) we support so far include
+
+* `moma_lasso()`: LASSO (least absolute shrinkage and selection operator).
+
+* `moma_scad()`: SCAD (smoothly clipped absolute deviation).
+
+* `moma_mcp()` MCP (minimax concave penalty).
+
+* `moma_slope()`: SLOPE (sorted L-one penalized estimation).
+
+* `moma_grplasso()`: Group LASSO.
+
+* `moma_fusedlasso()`: Fused LASSO.
+
+* `moma_spfusedlasso()`: Sparse fused LASSO.
+
+* `moma_l1tf()`: L-one trend filtering.
```

Comment:
You should be able to use MathJax / LaTeX here: $\ell_1$ and have it render properly.

Reply:
Fixed.